Automattic/jetpack
# Get Apps card: Iterate on link to jetpack.com/apps to ensure backwards compatibility and click-tracking

#### Changes proposed in this Pull Request:

This PR iterators on the link to `jetpack.com/apps` that can be found in the Get App card via _Jetpack_ → _Dashboard_. This includes the following changes:

* The [redirect helper](https://github.com/Automattic/jetpack/tree/trunk/projects/js-packages/components/tools/jp-redirect) is used to build the URL. Note, the instructions at PCYsg-pY7-p2 will be followed to set up the redirect once the naming proposed in this PR has been finalised/approved.
* An `onClick` function -- `trackAppLinkClick` -- has been added to the link to help us to track the times the links been clicked on. Similar to the above, the necessary internal processes to finish setting up the tracking will be done once we've agreed on the naming/code. 
* In addition, `rel="noopener noreferrer"` and `target="_blank"` have been added as values to the link, inline with the way similar links are handled throughout the Jetpack codebase. For example, see [this link](https://github.com/Automattic/jetpack/blob/510f1bf711062b55592ddcee7e5f6bee321c93b4/projects/plugins/jetpack/_inc/client/recommendations/sidebar/mobile-app/index.tsx#L85-L90) to the jetpack.com/mobile.

#### Other information:


#### Jetpack product discussion

N/A

#### Does this pull request change what data or activity we track or use?

Yes, this PR adds tracking for the times a user clicks on a link to the jetpack.com/apps page.

#### Testing instructions:

* Navigate to _Jetpack_ → _Dashboard_ to ensure this change doesn't alter the appearance of the apps card in any way.
* Note, the link won't functionally work until a redirect has been set up, which will happen once the naming in this PR has been finalised/approved.

<img width="1204" alt="Screenshot 2022-10-06 at 15 24 07" src="https://user-images.githubusercontent.com/2998162/194338768-4c30b74e-3a0f-4cc8-be5a-0f080c2fbf1b.png">
